pkg/collectors/feature_set.go

```diff
@@ -0,0 +1,69 @@
+package collectors
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/kube-state-metrics/pkg/metric"
+
+	"github.com/golang/glog"
+
+	v1 "github.com/openshift/api/config/v1"
+)
+
+var (
+	featureSetMetricFamilies = []metric.FamilyGenerator{
+		{
+			Name: "openshift_cluster_feature_set",
+			Type: metric.MetricTypeGauge,
+			Help: "Reports the feature set the cluster is configured to expose. name corresponds to the featureSet field of the cluster. The value is 1 if a cloud provider is supported.",
```

Comment:
s/name corresponds to the .../Name corresponds to the name of the feature gate./

Reply:
The last sentence doesn't seem like it applies to this metric, so I'd remove it.